internal/cloudapi/v2/handler.go

```diff
@@ -929,43 +955,65 @@ func (h *apiHandlers) GetComposeManifests(ctx echo.Context, id string) error {
 		return HTTPError(ErrorComposeNotFound)
 	}
 
-	// TODO: support non-koji builds
```

Comment:
One note here: manifests can contain secrets (most importantly activation keys) and this route isn't currently multi-tenant aware, which can lead to cross-tenant manifest (and thus secret) stealing. So before we can merge this, we need to make this route tenant-aware.

Reply:
I added appropriate middleware and multi-tenancy is now ensured for all `compose/<id>` endpoints...